### PR TITLE
8 post a new comment to an existing review

### PR DESCRIPTION
**Pull request by johnnyfwk**

### CODE REVIEW

File: src/components/SingleReviewPostComment.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                        ? <p id="review-comment-input-empty">Please enter a comment</p>
+                        : null}
+                {wasReviewCommentPostedSuccessfully === null
+                    ? null
+                    : wasReviewCommentPostedSuccessfully === true
+                        ? <p id="review-comment-posting-successful">Your comment was posted successfully!</p>
+                        : <p id="review-comment-posting-failed">Your comment was not posted.</p>}
+                {isCommentBeingPosted ? <p>Posting comment...</p> : null}
+
+				<input
```
Comment (popatre) [inline src/components/SingleReviewPostComment.jsx:54]: Could you change this is `textarea` to allow multi line responses?

File: src/components/SingleReviewPostComment.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    ? null
+                    : isReviewCommentInputEmpty === true
+                        ? <p id="review-comment-input-empty">Please enter a comment</p>
+                        : null}
+                {wasReviewCommentPostedSuccessfully === null
+                    ? null
+                    : wasReviewCommentPostedSuccessfully === true
+                        ? <p id="review-comment-posting-successful">Your comment was posted successfully!</p>
+                        : <p id="review-comment-posting-failed">Your comment was not posted.</p>}
+                {isCommentBeingPosted ? <p>Posting comment...</p> : null}
```
Comment (popatre) [inline src/components/SingleReviewPostComment.jsx:52]: Just as a general thing - nested ternaries are never ideally the way to go. Really hard to work out whats going on and maintain, if a change is needed. 
Could be clearer with just some if/else statements.

File: src/components/SingleReviewPostComment.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+				<input
+                    type="text"
+                    id="review-comment-input"
+                    name="review-comment-input"
```
Comment (popatre) [inline src/components/SingleReviewPostComment.jsx:57]: Nice controlled component

**Review by popatre — CHANGES_REQUESTED** [review]
Some nice code going on. Just a few things to add:

- [ ]  Would be better if the comment appeared at the top, at the moment, seems to randomly be in the middle of the comments. 
- [ ] Can you lock the post button after i click it. on a slow connection, i can keep clicking it whilst its posting, meaning i post multiple times. 
- [ ] Make the input a textarea so enable multiline stuff

- [ ] The comment number doesnt update after posting a comment

- [ ] Would be good to have some login functionality, even just a button that logs you in as the default (i see theres the state for this). Means you could add some functionality for when a user isnt logged in, they can't post. Could later build this out to a user login screen/selector.

File: src/components/SingleReviewPostComment.jsx [hunk 4]
```diff
@@ -51,14 +61,16 @@ export default function SingleReviewPostComment( {usernameLoggedIn, singleReview
...
                     id="review-comment-input"
                     name="review-comment-input"
                     value={reviewCommentInput}
                     onChange={onChangeReviewCommentInput}>
-                </input>
-                <button onClick={onClickSubmitCommentButton}>Submit Comment</button>
+                </textarea>				
+
+                {isCommentBeingPosted
+                    ? <button onClick={onClickSubmitCommentButton} disabled>Submit Comment</button>
```
Comment (P-Copley) [inline src/components/SingleReviewPostComment.jsx:72]: The conditions can be applied to the disabled attribute directly, you don't need to remove the whole element each time FYI 

e.g. `<button disabled={isCommentBeingPosted} ...`

**Comment by P-Copley** [comment]
Hey @johnnyfwk 👋 

Some great changes here. One small comment above but this is good to merge when you're ready 👍

**Pull request merged**